games/p.yaml

```diff
@@ -74,8 +74,7 @@
   - Buster Bros
   repo: http://pangzero.cvs.sourceforge.net/pangzero/pangzero/
   type: remake
-  updated: 2015-04-18
-  url: http://apocalypse.rulez.org/pangzero
```

Comment:
https://apocalypse.rulez.org/kozos/UpiPangBlog

Reply:
I changed it, but this blog is in Hungarian, might be difficult to understand by most visitors